hawc/apps/animal/managers.py

```diff
@@ -297,6 +298,105 @@ def migrate_terms(self, assessment: Assessment) -> pd.DataFrame:
 
         return df
 
+    def update_terms(self, objs: List[Dict]) -> List:
```

Comment:
Add a detailed comment to explain what this method does.

Also pass the assessment_id and add a validation check if the set of endpoint id assessments don't all equal the assessment id.